vumi/persist/txriak_manager.py

```diff
@@ -180,6 +184,16 @@ class TxRiakManager(Manager):
 
     call_decorator = staticmethod(inlineCallbacks)
 
+    def _setup_manager(self):
+        from twisted.internet import reactor
+        self._reactor = reactor
+        self._threadpool = ThreadPool(minthreads=2, maxthreads=2)
```

Comment:
Is 2 threads a good number? Is there a way we can set this in a config somewhere so that we can adjust it to see its effect?